docs/conf.py

```diff
@@ -334,6 +334,12 @@ def setup(app):
 theano.sandbox.cuda.cuda_enabled = True
 theano.sandbox.cuda.dnn.dnn_available = lambda: True
 
+theano.gpuarray = Mock()
+sys.modules['theano.gpuarray'] = theano.gpuarray
+sys.modules['theano.gpuarray.dnn'] = theano.gpuarray.dnn
+theano.gpuarray.pygpu_activated = True
+theano.gpuarray.dnn.dnn_present = lambda: True
+
```

Comment:
Technically, this should be a separate PR, but let's not overdo this, a separate commit will be good enough.